great_expectations/rule_based_profiler/parameter_builder/unexpected_count_statistics_multi_batch_parameter_builder.py

```diff
@@ -29,6 +27,8 @@
 from great_expectations.types.attributes import Attributes
 
 if TYPE_CHECKING:
+    import numpy as np
+
     from great_expectations.data_context.data_context.abstract_data_context import (
```

Comment:
@TrangPham We would like all of GX imports to be in the same section.  Thank you.